lib/teiserver_web/templates/admin/user/smurf_list.html.heex

```diff
@@ -346,27 +346,30 @@ is_moderator = allow?(@conn, "Moderator") %>
                   </tr>
                 <% end %>
 
-                <tr>
-                  <form
-                    action={~p"/teiserver/admin/users/create_smurf_key/#{@user.id}"}
-                    method="post"
-                  >
-                    <input type="hidden" name="_csrf_token" value={get_csrf_token()} />
-                    <td>
-                      <.input type="select" options={Map.keys(type_icons)} name="key" value="" />
-                    </td>
 
-                    <td>
-                      <.input type="text" name="value" value="" />
-                    </td>
+                <%= if allow?(@conn, "Server") do %>
```

Comment:
Tiny nitpick. At the top of the file there is already `is_moderator = allow?(@conn, "Moderator")`. You could do the same and reuse the value.
Although there are *also* other uses of `allow?(@conn, "Moderator")` elsewhere, so it's not super consistent to begin with.
I don't think there's a big impact either way since I believe the db calls are cached.